internal/oidc/token/token_handler.go

```diff
@@ -235,3 +235,16 @@ func getDownstreamUsernameFromPinnipedSession(session *psession.PinnipedSession)
 	}
 	return downstreamUsername, nil
 }
+
+func getDownstreamUsernameFromPinnipedSession(session *psession.PinnipedSession) (string, error) {
```

Comment:
The `git` diff for this function is weird.  Did you move it to a different file and then move it back?